Smartpad-macOS/Smartpad-macOS/Logic/ConnectionManager.swift

```diff
@@ -162,7 +178,9 @@ extension ConnectionManager{
      
         listVC.updateTable()
     }
-
+    
```

Comment:
Is this an intentional coding style or accidental? I've been reverting these blank lines but they keep coming back

Reply:
Not intentional!